bedrock/careers/templates/careers/base.html

```diff
@@ -0,0 +1,76 @@
+      data-gtm-code="{{ settings.GTM_ACCOUNT_CODE }}">
+  <head>
+    {# meta #}
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta property="og:description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <meta property="og:image" content="https://careers.mozilla.org{{ meta_image|default(static('img/careers/moz-m.png'), true)|safe }}">
+    <meta property="og:image:alt" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta name="twitter:title" content="{{ meta_title|default(default_slug, true)|safe }}">
+    <meta name="twitter:description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <meta name="twitter:image" content="https://careers.mozilla.org{{ meta_image|default(static('img/careers/moz-m.png'), true)|safe }}">
+    <meta name="twitter:card" content="summary">
+    <meta name="description" content="{{ meta_description|default(default_slug, true)|safe }}">
+    <title>{{ meta_title|default(default_slug, true)|safe }}</title>
+
+    {# css #}
+    {% block site_css %}
+      {{ css_bundle('careers') }}
+    {% endblock %}
+
+    {# js #}
+    <script src="{{ static('js/careers/dnt-helper.js') }}"></script>
+    <script src="{{ static('js/careers/protocol-base.min.js') }}"></script>
+    <script src="{{ static('js/careers/protocol-utils.min.js') }}"></script>
+    <script src="{{ static('js/careers/protocol-supports.min.js') }}"></script>
+
+    {# icons #}
+    <link rel="shortcut icon" type="image/png" href="{{ static('img/careers/favicon.ico') }}"/>
+
+    {# feeds #}
+    <link rel="alternate" type="application/rss+xml"
+          href="{{ url('careers.feed') }}" title="Current Mozilla job openings" />
+
+    {# TODO: Configure GA #}
+
+  </head>
+  <body id="{% block body_id %}{% endblock %}" class="{% block body_class %}{% endblock %}"
+    {%- block body_data %}
+    {%- endblock %}>
+    {%- if settings.GTM_ACCOUNT_CODE %}
+      <noscript>
+        <iframe src="//www.googletagmanager.com/ns.html?id={{ settings. GTM_ACCOUNT_CODE }}"
+                height="0" width="0" style="display:none;visibility:hidden"></iframe>
+      </noscript>
+    {%- endif %}
+
+    {% include "careers/includes/header.html" %}
+
+    <div id="wrapper" class="mzp-l-content mzp-t-mozilla">
+      <main role="main">
+
+        {% block content %}{% endblock %}
+
+      </main>
+    </div><!-- /#wrapper -->
+
+    {% block site_footer %}
+      {% include "careers/includes/footer.html" %}
+    {% endblock %}
+
+    {# js #}
+    {% block site_js %}
+      <script src="{{ static('js/careers/protocol-navigation.min.js') }}"></script>
+      <script src="{{ static('js/careers/protocol-menu.min.js') }}"></script>
```

Comment:
Most of the JS test/linting niggles are from these two ported-over-as-minified files. Am wondering if upgrading them to the latest version of `protocol` would be the quickest fix here, eg by just loading the ones the rest of bedrock uses? That's assuming there's no breaking change between the version used in `lumbergh` and the one currently in `bedrock`.

Reply:
There's a lot of stuff here that will likely need fixing. For now it's also possible to add `js/careers/` to .[eslintignore](https://github.com/mozilla/bedrock/blob/master/.eslintignore) and we can always make things good later on.

Reply:
Fwiw, I tested out trying to use the versions in bedrock, but it breaks the hamburger menu. So I think we'll need to stick to including these outdated versions for now until we can fix things.

Reply:
Thanks for the testing! I'll add the eslint ignore for now.

Reply:
I also added ignores for stylelint and prettier for these copied-over assets to get the front-end tests to pass.